tests/Feature/OrderCancellation/OrganisationWithoutTaxTest.php

```diff
@@ -69,19 +64,12 @@ public function cancels_and_refunds_order_with_single_ticket()
     public function cancels_and_refunds_order_with_multiple_tickets()
     {
         // Setup multiple attendee order but refund only 3 out of 5
-        list($order, $attendees) = $this->makeTicketOrder(5, 100.00);
+        [$order, $attendees] = $this->makeTicketOrder(5, 100.00);
         $attendeeIds = $attendees->pluck('id')->toArray();
         $response = $this->actingAs($this->getAccountUser())
             ->post("event/order/$order->id/cancel", [
-                'attendees' => [ $attendeeIds[0] ],
+                'attendees' => [$attendeeIds[0]],
```

Comment:
Sorry to be pedantic again. This change is breaking the test logic. In this test we are testing multiple attendees getting cancelled so please swap out the single attendee for the multiple like below:

```
                'attendees' => [
                    $attendeeIds[0],
                    $attendeeIds[1],
                    $attendeeIds[2],
                ],
```